bundle/jsky.app.ot/src/main/scala/edu/gemini/catalog/ui/QueryResultsFrame.scala

```diff
@@ -220,7 +220,28 @@ object QueryResultsFrame extends Frame with PreferredSizeFrame {
           doNameSearch(objectName.text)
       }
     }
-    lazy val instrumentName = new Label("")
+    lazy val instrumentBox = new ComboBox[SPComponentType](ObservationInfo.InstList.map(_._2)) with TextRenderer[SPComponentType] {
+      override def text(a: SPComponentType) = ~Option(a).flatMap(t => ObservationInfo.InstMap.get(t))
```

Comment:
It comes from scalaz, basically it gives you the "default" value if an option is `None`, for a `String` it would be the empty string